src/main/proto/ga4gh/bio_metadata_service.proto

```diff
@@ -0,0 +1,97 @@
+syntax = "proto3";
+
+package ga4gh;
+
+import "ga4gh/bio_metadata.proto";
+
+
+service BioMetadataService {
+  // Gets a list of Individuals accessible through the API.
+  // `POST /individuals/search` must accept a JSON version of
+  // `SearchIndividualsRequest` as the post body and will return a JSON version
+  // of `SearchIndividualsResponse`.
+  rpc SearchIndividuals(SearchIndividualsRequest)
+      returns (SearchIndividualsResponse);
+  
+  // `POST /biosamples/search` must accept a JSON version of
+  // `SearchBioSamplesRequest` as the post body and will return a JSON version
+  // of `SearchBioSamplesResponse`.
+  rpc SearchBioSamples(SearchBioSamplesRequest)
+      returns (SearchBioSamplesResponse);
+
+  // Gets an `Individual` by ID.
+  // `GET /individuals/{id}` will return a JSON version of `Individual`.
+  rpc GetIndividual(SearchIndividualsRequest)
+      returns (Individual);
+
+  // Gets a `BioSample` by ID.
+  // `GET /biosamples/{id}` will return a JSON version of `BioSample`.
+  rpc GetBioSample(SearchIndividualsRequest)
+      returns (BioSample);
+}
+// *********************  /individuals ********************************
+// This request maps to the body of `POST /individuals/search` as JSON.
+message SearchIndividualsRequest {
+  // The dataset to search within.
+  string dataset_id = 1;
```

Comment:
I believe all objects are should to be in datasets.  It is a
container around ownership.  It is also the only way we can
manage the scope of `name'.

Being in different datasets shouldn't be considered a barrier
using a object. If it is, the API has a serious problem.

Jerome Kelleher <notifications@github.com> writes:
> Why are we searching within a dataset here? There is no dataset_id attribute
> for Individuals or BioSamples, so it's not clear what this means. Should
> Individuals and BioSamples be considered to be "in" a dataset? I don't think
> so. I think these high-level metadata items should be repository level and
> useable within any dataset.
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub, or mute the thread.*
>